edge-middleware/feature-flag-split/package.json

```diff
@@ -10,6 +10,9 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@splitsoftware/splitio-browserjs": "^0.9.6",
+    "@splitsoftware/vercel-integration-utils": "1.0.0",
+    "@vercel/edge-config": "^0.2.1",
     "@vercel/examples-ui": "^1.0.5",
     "js-cookie": "^3.0.1",
     "next": "canary",
```

Comment:
Could you update this to the latest stable?

Reply:
It seems like we're using `canary` in 53 cases and `latest` in only 14 that's why I went with canary. Is the guidance to always use latest?